### ISSUE TITLE
Instructions for setting up run environment in the "How to Use" section

### ISSUE THREAD

**Issue by brian-rose** [ProjectPythia/pythia-foundations#198]
As discussed in #127 and at several working group meetings, we need to add some support for users who want to download and run individual notebooks without cloning the entire repository.

The instructions should include a command like
```
conda env create -f https://raw.githubusercontent.com/ProjectPythia/pythia-foundations/main/environment.yml
```
for creating the `pythia-book-dev` environment, along with additional instructions for activating and updating that environment.

**Comment by brian-rose** [comment]
We may possibly want to add some boilerplate instructions in every notebook giving a link back to the "How to Use" page for setting up a run environment.

**Comment by brian-rose** [comment]
Closing this as I believe we addressed this in #199.
